package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
If we anyhow rework this file, might be good to add the `files` and `types` properties. Types to explicitly import types where needed. Files so it will publish to npm only the code which is relevant for the consumer and not the `example` folder etc.
If you move the core files to src folder I would put it as:
```
"files": [
  "./src",
  "./index.d.ts",
  "README.md"
]
```

Or in current structure:
```
"files": [
  "./index.js",
  "./browser.js",
  "./seq_logger.js",
  "./index.d.ts",
  "./README.md"
]
```